test/test.ts

```diff
@@ -832,7 +834,7 @@ it('pkg_support', async () => {
 });
 
 // `go1.x` runtime, requires go@1.15, must not be a higher version
-it(
+it.skip(
```

Comment:
I don't think we should skip this without a better understanding of what's happening.

Reply:
I briefly looked into this yesterday and found that newer versions of `go` (1.20) appear to require a `go.mod` which this test does not have. I tried adding one and it would fail to download some ancient dependency.

I then tried `go@1.15`, but it fails with `LambdaError: RequestId: ba992ae3-eba7-45e3-818d-409762d6452d Process exited before completing request`. I have no idea why the request doesn't complete properly.